### PR TITLE
Add 'Porkbun' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22porkbun%22%2C%22relevant-countries%22%3A%5B%22all%22%5D%2C%22categories%22%3A%5B%22telecommunication%22%5D%2C%22name%22%3A%22Porkbun%22%2C%22address%22%3A%2221370%20SW%20Langer%20Farms%20Parkway%2C%20Suite%20142-429%5CnSherwood%2C%20OR%2097140%2C%20US%22%2C%22email%22%3A%22abuse%40porkbun.com%22%2C%22web%22%3A%22https%3A%2F%2Fporkbun.com%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fporkbun.com%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1324%22%5D%2C%22required-elements%22%3A%5B%7B%22desc%22%3A%22Account%20E-Mail%20Address%22%2C%22type%22%3A%22email%22%2C%22optional%22%3Afalse%7D%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/porkbun.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "porkbun",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication"
+    ],
+    "name": "Porkbun",
+    "address": "21370 SW Langer Farms Parkway, Suite 142-429\nSherwood, OR 97140, US",
```
Comment (moriarty-jim) [inline companies/porkbun.json:10]:
```suggestion
    "address": "21370 SW Langer Farms Parkway, Suite 142-429\nSherwood, OR 97140\nUnited States",
```
fixed address format

File: companies/porkbun.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "email": "abuse@porkbun.com",
```
Comment (moriarty-jim) [inline companies/porkbun.json:11]:
```suggestion
    "email": "privacy@porkbun.com",
```
maybe better email?

File: companies/porkbun.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "web": "https://porkbun.com",
+    "sources": [
+        "https://porkbun.com",
```
Comment (moriarty-jim) [inline companies/porkbun.json:14]:
```suggestion
        "https://porkbun.com/legal/agreement/privacy_policy",
```

File: companies/porkbun.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        "https://github.com/datenanfragen/data/pull/1324"
+    ],
+    "required-elements": [
+        {
+            "desc": "Account E-Mail Address",
```
Comment (moriarty-jim) [inline companies/porkbun.json:19]:
```suggestion
            "desc": "Email",
```
naming according to #720 preferred?

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
some suggestions/fixes

File: companies/porkbun.json (hunk 1 above)
Comment (baltpeter) [reply companies/porkbun.json:10]: Almost, we've decided on "United States of America". :)

```suggestion
    "address": "21370 SW Langer Farms Parkway\nSuite 142-429\nSherwood, OR 97140\nUnited States of America",
```

**Review by baltpeter — APPROVED** [review]
Thanks, everyone.

**Pull request merged**